util/b64encode.c

```diff
@@ -97,7 +97,7 @@ b64decode(const char * in, size_t inlen, uint8_t * out, size_t * outlen)
 
 		/* Output 3 bytes. */
 		for (i = 0; i < 3; i++) {
-			out[i] = t >> 16;
+			out[i] = (uint8_t) (t >> 16);
```

Comment:
Unnecessary whitespace in cast.  And I'd prefer `(t >> 16) & 0xff` if that silences the warning.